gpt4all-chat/localdocsmodel.h

```diff
@@ -31,12 +31,19 @@ class LocalDocsModel : public QAbstractListModel
         InstalledRole,
         IndexingRole,
         ErrorRole,
+        ForceIndexingRole,
         CurrentDocsToIndexRole,
         TotalDocsToIndexRole,
         CurrentBytesToIndexRole,
         TotalBytesToIndexRole,
         CurrentEmbeddingsToIndexRole,
-        TotalEmbeddingsToIndexRole
+        TotalEmbeddingsToIndexRole,
+        TotalDocsRole,
+        TotalWordsRole,
+        TotalTokensRole,
+        LastUpdateRole,
+        FileCurrentlyProcessingRole,
+        EmbeddingModelRole
```

Comment:
nit: I prefer trailing commas on enums

Reply:
The code is consistent with other code that preceded.